gotham_reqwest/src/main.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2020 O.S. Systems Sofware LTDA
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use std::sync::{Arc, Mutex};
+#[macro_use]
+extern crate gotham_derive;
```

Comment:
This can be imported as:
```
use gotham_derive::StateData;
```

in the serve function to keep the gotham imports together.